projects/schematics/src/migrations/5_0/constructor-deprecations/data/configurator-attribute-numeric-input-field.component.migration.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  CONFIGURATOR_ATTRIBUTE_NUMERIC_INPUT_FIELD_COMPONENT,
+  CONFIGURATOR_ATTRIBUTE_NUMERIC_INPUT_FIELD_SERVICE,
+  CONFIGURATOR_UI_SETTINGS_CONFIG,
+  TRANSLATION_SERVICE,
+} from '../../../../shared/constants';
+import {
+  SPARTACUS_CORE,
+  SPARTACUS_PRODUCT_CONFIGURATOR_RULEBASED,
+} from '../../../../shared/libs-constants';
+import { ConstructorDeprecation } from '../../../../shared/utils/file-utils';
+
+export const CONFIGURATOR_ATTRIBUTE_INPUT_FIELD_COMPONENT_MIGRATION: ConstructorDeprecation =
```

Comment:
Please have this constant named according to the underlying component and the migration file name:
```suggestion
export const CONFIGURATOR_ATTRIBUTE_NUMERIC_INPUT_FIELD_COMPONENT_MIGRATION: ConstructorDeprecation =
```